docs/en/relying-party-solution.rst

```diff
@@ -745,3 +746,419 @@ The Entity Configuration is a JWS, where its header parameters are defined below
     The Relying Party specific metadata parameter are experimental 
     and still under discussion `here <https://github.com/openid/OpenID4VP/issues/17>`_.
 
+
+Proximity Flow
+==============
+
+This section describes how a Relying Party requests the presentation of an *mDoc-CBOR* Credential to a Wallet Instance according to the *ISO 18013-5 Specification*. Only *Supervised Device Retrieval flow* is supported in this technical implementation profile. 
+
+The presentation phase is divided into three sub-phases: 
+ 
+  1. **Device Engagement**: This subphase starts when the User is asked to disclose some attributes from their mDoc(s). The objective of this subphase is to establish a secure communication channel between the Wallet Instance and the Verifier App, so that the mDoc requests and responses can be exchanged during the communication subphase.
```

Comment:
```suggestion
  1. **Device Engagement**: This subphase begins when the User is prompted to disclose certain attributes from the mDoc(s). The objective of this subphase is to establish a secure communication channel between the Wallet Instance and the Verifier App, so that the mDoc requests and responses can be exchanged during the communication subphase.
```